source/grid/grid_generator.cc

```diff
@@ -3004,6 +3003,58 @@ namespace GridGenerator
     tria.set_manifold(0, SphericalManifold<2>(center));
   }
 
+  template <int dim>
```

Comment:
Please use three blank lines between definitions.

Reply:
This should also be resolved in the last commit. Sorry, I don't know how show the update.